requirements.txt

```diff
@@ -1,3 +1,4 @@
 Django>=4.0.4,<4.1
 djangorestframework>=3.13.1,<3.14
-psycopg2>=2.9.3,<2.10
\ No newline at end of file
+psycopg2>=2.9.3,<2.10
+pydantic>=1.8.2
```

Comment:
Often we'd lock to a specific version to save an update breaking our code unexpectedly. So `pydantic==1.8.2` for example